x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/execution/search/extractor/HitExtractor.java

```diff
@@ -19,6 +19,11 @@ public interface HitExtractor extends NamedWriteable {
      */
     Object extract(SearchHit hit);
 
+    /**
+     * Extract the value from a hit.
+     */
+    Object extract(SearchHit hit, boolean allowMultivalue);
```

Comment:
It isn't clear what this method brings since `allowMultivalue` parameter is ignored in most implementations which just delegate to `extract(SearchHit)`